### PR TITLE
#733 react-scripts: update to 3.0.1

### PR DESCRIPTION
**Pull request by martinfrances107**
## Issue

https://github.com/jsdrupal/drupal-admin-ui/issues/733

## Screenshot / UI changes

Describe the UI changes / make a screenshot

## Testing instructions

Please describe with a few words how to reproduce the problem / test the feature

I want to upgrade to the latest create-react-app 

This means bumping the package versions of 

react-scripts
eslint

### CODE REVIEW

File: packages/components/src/Components/FormElements/InputCheckbox/InputCheckbox.js [hunk 1]
```diff
@@ -142,25 +142,33 @@ const InputCheckbox = props => {
 };
 
 InputCheckbox.propTypes = {
+  checked: PropTypes.bool,
```
Comment (dawehner) [inline packages/components/src/Components/FormElements/InputCheckbox/InputCheckbox.js]: In general, try to make these changes in a separate PR, they aren't really related with updating create-react-app :)

**Comment by martinfrances107** [comment]
Thanks for looking at this

Ah the test would not pass unless changes to the prop types were made

Like a game of dominoes 

changing the react-scripts triggered a change in linter. etc etc.
 
Maybe the changes to default settings ... could have been done in a separate issue.

**Comment by dawehner** [comment]
I totally get this. The changes of the proptypes you can though still do in a separate PR and make it a dependency of some other one.

**Comment by martinfrances107** [comment]
Ah I see what you mean ... I should have spawned 3ish child issues...I will adjust my approach in future.

**Comment by dawehner** [comment]
Thank you Martin.

I always try to assume that the person reviewing has minimal time, so I try
to make their life easier. I make mistakes all the time, but it's a good
thing to keep in mind.
